paper-button.css

```diff
@@ -0,0 +1,46 @@
+:host {
+  display: inline-block;
+  position: relative;
+  box-sizing: border-box;
+  min-width: 5.14em;
+  margin: 0 0.29em;
+  background: transparent;
+  text-align: center;
+  font: inherit;
+  text-transform: uppercase;
+  outline: none;
+  border-radius: 3px;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  -webkit-user-select: none;
+  user-select: none;
+  cursor: pointer;
+  z-index: 0;
+}
+
+:host([disabled]) {
+  background: #eaeaea;
+  color: #a8a8a8;
+  cursor: auto;
+  pointer-events: none;
+}
+
+:host([noink]) paper-ripple {
+  display: none;
+}
+
+paper-card {
+  border-radius: inherit;
+}
+
+/*:host(:not([raised])) paper-card {*/
+  /*box-shadow: none !important;*/
+/*}*/
+
+::content * {
```

Comment:
[`::content` needs to wrapped](https://github.com/Polymer/polymer/blob/0.8-preview/PRIMER.md#scoped-styling) to work in shady because it gets removed. The selector will end up being `*` in this case.

```
.content > ::content * {
  ...
}

...

<div class="content">
  <content></content>
</div>
```